packages/focusvisible/src/FocusVisibleContainer.spec.tsx

```diff
@@ -35,7 +35,10 @@ describe('FocusVisibleContainer', () => {
 
     expect(() => {
       const ErrorExample = () => {
+        /* eslint-disable @typescript-eslint/ban-ts-ignore */
+        // @ts-ignore
```

Comment:
Could you also include a comment on why we are ignore ts errors here? Something along the lines off "Ignoring to test JS runtime usage"

Reply:
Yes, that is a great idea. Thanks for the suggestion.